components/CheckDocumentation/vacuum/XminHorizon.tsx

```diff
@@ -0,0 +1,342 @@
+import SQL from "../../SQL";
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+} from "../../../util/checks";
+import { useCodeBlock } from "../../CodeBlock";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const XminHorizonTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const threshold = config.settings["behind_hours"] as number;
+  const hourPluralized = threshold === 1 ? "hour" : "hours";
+  return (
+    <p>
+      Detects when the xmin horizon on the server was assigned at more than{" "}
+      <code>{threshold}</code> {hourPluralized} ago and creates an issue with
+      severity "info". The issue will be created even if no VACUUM is currently
+      blocked by this, as this will potentially block any future VACUUMs.
+      Resolves once the xmin horizon is no longer behind.
```

Comment:
Does "no longer behind" mean "under the threshold" here?

Reply:
Hmm, yeah, I suppose. Maybe "once the age of the xmin horizon is under the threshold"? I think using "once the xmin horizon is under the threshold" as a shorthand also sounds okay to me if you think the above is too verbose.

Reply:
I'll go with "once the age of the xmin horizon is under the threshold" 👍

Reply:
🤦 just realized that I forgot to update the "is under the threshold" part. I'm going to make a follow-up PR

Reply:
https://github.com/pganalyze/pganalyze-docs/pull/162
Decided to make even longer, making it "assignment age". This matches well with the graph title.